deployments/mainnet/usdc/migrations/1659582050_raise_supply_caps_and_seed_reserves_and_rewards.ts

```diff
@@ -0,0 +1,96 @@
+import { DeploymentManager, migration } from '../../../../plugins/deployment_manager';
+import { calldata, exp, proposal } from '../../../../src/deploy';
+
+const comptrollerAddress = '0x3d9819210a31b4961b30ef54be2aed79b9c9cd3b';
+
+export default migration('1659582050_raise_supply_caps_and_seed_reserves_and_rewards', {
+  prepare: async (deploymentManager: DeploymentManager) => {
+    return {};
+  },
+
+  enact: async (deploymentManager: DeploymentManager) => {
+    const trace = deploymentManager.tracer();
+    const ethers = deploymentManager.hre.ethers;
+
+    const {
+      governor,
+      comet,
+      configurator,
+      cometAdmin,
+      rewards,
+      USDC,
+      COMP,
+      WBTC,
+      WETH,
+      UNI,
+      LINK
+    } = await deploymentManager.getContracts();
+
+    const actions = [
+      // 1. Increase supply caps for each of the assets
+      {
+        contract: configurator,
+        signature: "updateAssetSupplyCap(address,address,uint128)",
+        args: [comet.address, COMP.address, exp(150_000, 18)],
+      }, {
+        contract: configurator,
+        signature: "updateAssetSupplyCap(address,address,uint128)",
+        args: [comet.address, WBTC.address, exp(2_000, 8)],
+      }, {
+        contract: configurator,
+        signature: "updateAssetSupplyCap(address,address,uint128)",
+        args: [comet.address, WETH.address, exp(25_000, 18)],
+      }, {
+        contract: configurator,
+        signature: "updateAssetSupplyCap(address,address,uint128)",
+        args: [comet.address, UNI.address, exp(250_000, 18)],
```

Comment:
This one looks off. UNI is ~$8.5 right now (same price as LINK basically), so supply cap should be around 1mn as well.

Reply:
Updated to 1M UNI